src/deployment-guides/resources/tko-on-aws/tkg_vpc/config_files/grafana-data-values.yaml

```diff
@@ -0,0 +1,85 @@
+---
+#! The namespace in which to deploy grafana.
+namespace: tanzu-system-dashboards
+
+grafana:
+  #! Grafana deployment configuration
+  deployment:
+    replicas: 1
+    containers:
+      resources: {}
+    podAnnotations: {}
+    podLabels: {}
+    k8sSidecar:
+      containers:
+        resources: {}
+  #! Grafana service configuration
+  service:
+    type: LoadBalancer
+    port: 80
+    targetPort: 3000
+    labels: {}
+    annotations: {}
+  #! The grafana configuration.
+  config:
+    grafana_ini: |
+      [analytics]
+      check_for_updates = false
+      [grafana_net]
+      url = https://grafana.com
+      [log]
+      mode = console
+      [paths]
+      data = /var/lib/grafana/data
+      logs = /var/log/grafana
+      plugins = /var/lib/grafana/plugins
+      provisioning = /etc/grafana/provisioning
+    #! Refer to https://grafana.com/docs/grafana/latest/administration/provisioning/#example-data-source-config-file
+    datasource_yaml: |-
+      apiVersion: 1
+      datasources:
+        - name: Prometheus
+          type: prometheus
+          url: prometheus-server.tanzu-system-monitoring.svc.cluster.local
+          access: proxy
+          isDefault: true
+    #! Refer to https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards
+    dashboardProvider_yaml: |-
+      apiVersion: 1
+      providers:
+        - name: 'sidecarDashboardProvider'
+          orgId: 1
+          folder: ''
+          folderUid: ''
+          type: file
+          disableDeletion: false
+          updateIntervalSeconds: 10
+          allowUiUpdates: false
+          options:
+            path: /tmp/dashboards
+            foldersFromFilesStructure: true
+  #! Grafana pvc configuration
+  pvc:
+    annotations: {}
+    storageClassName: null
+    accessMode: ReadWriteOnce
+    storage: "2Gi"
+  secret:
+    type: "Opaque"
+    admin_user: "YWRtaW4="
```

Comment:
same here.. we need to generate or require as input

Reply:
these are the sample password , user should enter real password and other values. i added sample password and all mandatory fields into to these template files with sample values  , otherwise they will throw validation errors during package creation. instructions given into read me file